app/code/Bulbulatory/Recommendations/etc/db_schema.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="bulbulatory_recommendations" resource="default" engine="innodb" comment="Bulbulatory Recommendations">
+        <column xsi:type="int" name="id" padding="10" unsigned="true" nullable="false" identity="true" comment="Affiliate link ID"/>
+        <column xsi:type="text" name="reference" comment="Customer Reference"/>

```

Comment:
Will it refer to the customer ID? Or customer email? Or maybe something else? Check out the `customer_entity` table and maybe try to create a foreign key that references the `customer_entity` table.

Reply:
@TM18

> Will it refer to the customer ID? Or customer email? Or maybe something else? Check out the customer_entity table and maybe try to create a foreign key that references the customer_entity table.

I misunderstood this field description. I thought that _customer reference_ - it's Recommendations Comment (like "It's very good site. I recommend it to you!"). I created new filed _customer_id.bulbulatory_recommendations__ that I tied with table 
 _customer_entity_ by _foreign key_